plugins/DuskVerb/tools/tuner/tune_preset.py

```
@@ -0,0 +1,218 @@
+           "--param", "Dry/Wet=1.0",
+           "--param", "Bus Mode=1",
+           "--param", "Freeze=0"]
+    for k, v in params.items():
+        if k in {"Dry/Wet", "Bus Mode", "Freeze"}:
+            continue
+        cmd += ["--param", f"{k}={v}"]
+    if extra:
+        for k, v in extra.items():
+            cmd += ["--param", f"{k}={v}"]
+    cmd.append(preset)
+    r = subprocess.run(cmd, capture_output=True, text=True, timeout=180)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)